Applications/Spire/Include/Spire/Spire/SpinBoxModel.hpp

```diff
@@ -0,0 +1,123 @@
+#ifndef SPIRE_SPIN_BOX_MODEL_HPP
+#define SPIRE_SPIN_BOX_MODEL_HPP
+#include <QHash>
+#include "Nexus/Definitions/Money.hpp"
+#include "Nexus/Definitions/Quantity.hpp"
+#include "Spire/Ui/CustomQtVariants.hpp"
+#include "Spire/Ui/RealSpinBox.hpp"
+
+namespace Spire {
+
+  //! Represents the data model for a spin box.
+  template <typename T>
+  class SpinBoxModel {
+    public:
+
```

Comment:
Don't reference ```T``` within the template, instead reify the parameter with a typedef as follows:

    using Type = T;

Then refer to ```Type```.